modules/common/condition/funcs_test.go

```diff
@@ -387,10 +387,10 @@ func TestMirror(t *testing.T) {
 	g.Expect(conditions).To(haveSameConditionsOf(CreateList(unknownReady, trueA)))
 	targetCondition = conditions.Mirror("targetConditon")
 	// expect to be mirrored trueA
```

Comment:
this comment should be updated to unknownReady, right?

Reply:
you are right, I need to fix this